src/Components/QuestionEdit/QuestionEdit.tsx

```diff
@@ -22,7 +23,16 @@ export const QuestionEdit = () => {
         <Form
             onSubmit={(e) => {
                 e.preventDefault();
-                addQuestion(questionEditValues);
+                let questionIndex;
+                let findIndex = questions?.find(({ id }, index) => {
+                    if (id === questionId) questionIndex = index;
+                })?.id;
+
+                if (questionIndex !== undefined) {
+                    questions[questionIndex] = questionEditValues;
+
+                    updateQuestion(questions);
+                }
```

Comment:
this is to update questions but IMO it's a nasty fix, I will try to refactor it but I would love some input on how I could improve it.